include/proj/coordinateoperation.hpp

```diff
@@ -155,10 +155,19 @@ class PROJ_GCC_DLL CoordinateOperation : public common::ObjectUsage,
 
     PROJ_DLL CoordinateOperationNNPtr normalizeForVisualization() const;
 
-    PROJ_PRIVATE :
-        //! @cond Doxygen_Suppress
-        PROJ_FOR_TEST CoordinateOperationNNPtr
-        shallowClone() const;
+  PROJ_PRIVATE :
+    //! @cond Doxygen_Suppress
+    PROJ_FOR_TEST CoordinateOperationNNPtr
+    shallowClone() const;
+
+    PROJ_INTERNAL void setCRSs(const crs::CRSNNPtr &sourceCRSIn,
```

Comment:
what justifies setCRS and setProperties to be public/PROJ_PRIVATE and no longer protected ?